service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -119,9 +121,20 @@ public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
         Placement currentPlacement = placementService.getPlacement(caseData, child);
         Placement previousPlacement = placementService.getPlacement(caseDataBefore, child);
 
+        // TODO refactor: this logic is confusing. Suggested: new method isNewOrUpdated...
         if (!isUpdatingExistingPlacement(previousPlacement, currentPlacement)) {
             publishPlacementApplicationUploadEvent(callbackRequest);
```

Comment:
Do you think there is a way you could reuse this method? Maybe change the parameter to be a `CallbackEvent` and the method just sends the notification instead of building the object as well.

Reply:
how do you imagine this? checking if new or edited placement is different to the UUID check I am doing

Reply:
🤷‍♂